plugins/SitesManager/Controller.php

```diff
@@ -335,4 +337,43 @@ private function getVueInitializeCode($vueVersion = '3')
 window._paq.push(['trackPageView']); // To track a page view
 INST;
     }
+
+    private function mergeMultipleNotification(&$templateData)
+    {
+        $isNotificationsMerged = false;
+        $bannerMessage = '';
+        $guides = [];
+
+        if ($templateData['ga3Used'] || $templateData['ga4Used']) {
+            $bannerMessage = 'Google Analytics ';
+            $ga3GuideUrl =  '<a href="https://matomo.org/faq/how-to/migrate-from-google-analytics-3-to-matomo/" target="_blank" rel="noreferrer noopener">Google Analytics 3</a>';
+            $ga4GuideUrl =  '<a href="https://matomo.org/faq/how-to/migrate-from-google-analytics-4-to-matomo/" target="_blank" rel="noreferrer noopener">Google Analytics 4</a>';
+            if ($templateData['ga3Used'] && $templateData['ga4Used']) {
+                $isNotificationsMerged = true;
+                $guides[] = $ga3GuideUrl;
+                $guides[] = $ga4GuideUrl;
+                $bannerMessage .= '3 & 4';
+            } else {
+                $bannerMessage .= ($templateData['ga3Used'] ? 3 : 4);
+                $guides[] .= ($templateData['ga3Used'] ? $ga3GuideUrl : $ga4GuideUrl);
+            }
+        }
+
+        if ($bannerMessage && $templateData['consentManagerName']) {
+            $isNotificationsMerged = true;
+            $bannerMessage .= ' and ' . $templateData['consentManagerName'];
```

Comment:
This way, if there's no GA at all, would the message start with " and "? As in...
>  and Osano were detected on your website. Learn more about configuring Matomo..

Reply:
Apologies, I'm not trying to be difficult here, it just doesn't look 100% bullet proof.

Reply:
it won't start with `and Osano`

Reply:
![Screenshot from 2023-06-01 11-50-16](https://github.com/matomo-org/matomo/assets/14902277/c83b50e1-cc26-4b9a-8432-58978d1528e2)

Reply:
@michalkleiner I have added additional UI tests to make it clear and we already have test for [consent manager](https://github.com/matomo-org/matomo/blob/5.x-dev/tests/UI/specs/EmptySite_ConsentManager_spec.js)

Reply:
Anyway. The word `and` is untranslated here 🙈 
And some languages might actually not use `,` or even an `and` in listings. This is actually something we hadn't considered in the past. The i18n part of Matomo currently doesn't cover this. I will have a quick look at that and check if we can add that to our Intl plugin for easy usage.

Reply:
I have updated to use Translations for now, will change to above PR way once its merged